corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -46,18 +42,25 @@
                   "><i class="fa fa-remove"></i></button>
         </legend>
       </div>
-      <div data-bind="if: collapsible" style="display: flex">
-        <div class="collapsible-icon-container" style="align-self: center">
-          <i style="" class="fa" data-bind="
-              css: {'fa-angle-double-right': !showChildren(), 'fa-angle-double-down': showChildren()},
-          "></i>
-        </div>
-        <span class="webapp-markdown-output caption" data-bind="html: caption_markdown() || caption(), attr: {id: captionId()}"></span>
-        <i class="fa fa-warning text-danger pull-right" data-bind="visible: hasError() && !showChildren()"></i>
-        <button class="btn btn-danger del pull-right" href="#" data-bind="
-                    visible: isRepetition,
-                    click: deleteRepeat
-                "><i class="fa fa-remove"></i></button>
+      <div data-bind="if: collapsible" >
+        <div
+            style="display: flex"  data-bind="
+          style: {'background-color': headerBackgroundColor(),
+            'color': headerBackgroundColor() ? 'white' : '',
+            'margin': '0 23px 0 23px',
+        }">
+          <div class="collapsible-icon-container" style="align-self: center">
+            <i style="margin: 10px" class="fa" data-bind="
+                css: {'fa-angle-double-right': !showChildren(), 'fa-angle-double-down': showChildren()},
+            "></i>
+          </div>
+          <span style="align-self: center" class="webapp-markdown-output caption" data-bind="html: caption_markdown() || caption(), attr: {id: captionId()}"></span>
```

Comment:
Would Bootstrap's `text-center` or `center-block` class work here instead of `align-self: center`?

Reply:
they are for horizontally alignment. From bootstrap 4 on there will be `d-flex` and  `align-items-center` so I recreated them for now.